pyneuroml/plot/PlotMorphologyVispy.py

```diff
@@ -238,6 +254,29 @@ def vispy_on_key_press(event):
         elif event.text == "9":
             canvas.app.quit()
 
+    def start_recording(canvas, output_file):
+        nonlocal recording, frames
+        recording = True
+        frames = []
+        print(f"Recording started. Output file: {output_file}")
+
+    def stop_recording(canvas):
+        nonlocal recording, frames
+        recording = False
+        output_file = f"output-{len(glob.glob('output-*.mp4'))}.mp4"
```

Comment:
we generated the name of the output file above already---can we not re-use that here also?